.github/taskfiles/docs.yml

```
@@ -0,0 +1,148 @@
+    internal: true
+
+tasks:
+  test:
+    desc: Run all tests
+    cmds:
+      - task build:install -s
+      - printf "${BLUE}[INFO] Running tests...${RESET}\n"
+      - |
+        # Find source folder
+        SOURCE_FOLDER="src/$(find src -mindepth 1 -maxdepth 1 -type d -not -path '*/\.*' | head -1 | sed 's|^src/||')"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)